qcdb/intf_nwchem/options.py

```diff
@@ -704,66 +750,232 @@ def load_nwchem_defaults(options):
                 glossary= 'Number of orbitals to freeze'))  # expand to core/atomic/virtual
 
     #CCSD(T) block
-#    options.add(
-#            'nwchem',
-#            RottenOption(
-#                keyword= 'ccsd(t)_maxiter',
-#                default= 20,
-#                validator= parsers.positive_integer,
-#                glossary= 'Maximum numbers of iterations; iterations default is 20.'))
-#    options.add(
-#            'nwchem',
-#            RottenOption(
-#                keyword='ccsd(t)__thresh',
-#                default= 1.0e-6,
-#                validator=parsers.parse_convergence,
-#                glossary= 'Convergence threshold for the iterative part of the calculation.'))
+    options.add(
+            'nwchem',
+            RottenOption(
+                keyword= 'ccsd_pr_t__maxiter',
```

Comment:
I think the maxiter for ccsd(t) is handled in the CCSD block, so these shouldn't be needed. No iterations associated with the (T) step.